com.github.stsdc.monitor.json

```diff
@@ -0,0 +1,347 @@
+{
+  "app-id": "com.github.stsdc.monitor",
+  "runtime": "io.elementary.Platform",
+  "runtime-version": "7.3",
+  "sdk": "io.elementary.Sdk",
+  "command": "com.github.stsdc.monitor",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=session-bus",
+    "--socket=system-bus",
+    "--device=shm",
+    "--device=all",
+    "--share=network",
+    "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/usr/share:/run/host/share:/var/lib/flatpak/exports/share:~/.local/share/flatpak/exports/share",
+    "--talk-name=org.freedesktop.Flatpak",
+    "--talk-name=com.github.stsdc.monitor",
+    "--system-talk-name=org.freedesktop.Accounts",
+    "--system-talk-name=org.freedesktop.UDisks2",
+    "--filesystem=host:ro",
+    "--filesystem=/var/lib/flatpak/app:ro",
+    "--filesystem=/var/lib/flatpak/exports/share:ro",
+    "--filesystem=xdg-data/flatpak/app:ro",
+    "--filesystem=xdg-data/flatpak/exports/share:ro",
+    "--filesystem=/run/docker.sock",
+    "--filesystem=xdg-run/docker.sock",
+    "--filesystem=/run/podman/podman.sock",
+    "--filesystem=xdg-run/podman/podman.sock",
+    "--filesystem=/tmp"
```

Comment:
I have so many questions about these filesystem permissions. Notably with `/var/lib/flatpak`, why `xdg-data`, etc.

Reply:
Getting access to the podman and docker sockets. Precedent: https://github.com/flathub/com.github.sdv43.whaler/blob/master/com.github.sdv43.whaler.yml#L10-L13

Reply:
two wrongs don't make a right.

Reply:
The app lists running containers and their resources.